### PR TITLE
Remove aws

### PR DESCRIPTION
**Pull request by physX23**
https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/98111

### CODE REVIEW

File: bin/appcenter.sh [hunk 1]
```diff
@@ -67,10 +63,10 @@ if [ "${2}" != "" ]; then
   echo "${2}" > .xtc/device-agent-sha
 fi)
 
-S3_ROOT="https://s3-eu-west-1.amazonaws.com/calabash-files/dylibs/to-test-injection"
-LIB_BEETS="${S3_ROOT}/libBetaVulgaris.dylib"
-LIB_CABBAGE="${S3_ROOT}/libBrassica.dylib"
-LIB_CUCUMBER="${S3_ROOT}/libCucurbits.dylib"
+AZURE_ROOT="./files"
```
Comment (igorovsiannikov) [inline bin/appcenter.sh]: is it really Azure root? or it's just libraries root directory?

File: bin/appium/xtc-submit.sh [hunk 2]
```diff
@@ -63,10 +63,10 @@ if [ "${SERIES}" = "" ]; then
 fi
 
 PIPELINE="pipeline:detect-dylibs-to-inject-in-app-bundle"
-S3_ROOT="https://s3-eu-west-1.amazonaws.com/calabash-files/dylibs/to-test-injection"
-LIB_BEETS="${S3_ROOT}/libBetaVulgaris.dylib"
-LIB_CABBAGE="${S3_ROOT}/libBrassica.dylib"
-LIB_CUCUMBER="${S3_ROOT}/libCucurbits.dylib"
+AZURE_ROOT="../files"
```
Comment (igorovsiannikov) [inline bin/appium/xtc-submit.sh]: same

File: bin/xcuitest/xtc-submit.sh [hunk 3]
```diff
@@ -69,10 +69,10 @@ if [ "${SERIES}" = "" ]; then
 fi
 
 PIPELINE="pipeline:<branch>"
-S3_ROOT="https://s3-eu-west-1.amazonaws.com/calabash-files/dylibs/to-test-injection"
-LIB_BEETS="${S3_ROOT}/libBetaVulgaris.dylib"
-LIB_CABBAGE="${S3_ROOT}/libBrassica.dylib"
-LIB_CUCUMBER="${S3_ROOT}/libCucurbits.dylib"
+AZURE_ROOT="../files"
```
Comment (igorovsiannikov) [inline bin/xcuitest/xtc-submit.sh]: also same

File: bin/ci/appcenter.sh [hunk 4]
```diff
@@ -18,48 +18,26 @@ else
...
-fi
-
 IPA=Products/ipa/DeviceAgent/DeviceAgent-Runner.ipa
 SHA=$(/usr/bin/shasum "${IPA}" | awk '{print $1}')
 info "Will use DeviceAgent.iOS: ${SHA}"
 
 # Push will succeed even if file already exists
-URL="s3://calabash-files/ios-device-agent/${SHA}/DeviceAgent-Runner.ipa"
-aws s3 cp "${IPA}" "${URL}"
+az storage blob upload \
```
Comment (sergei-fedorov) [inline bin/ci/appcenter.sh:26]: Did you test DeviceAgent build and upload works as before?

**Review by sergei-fedorov — APPROVED** [review]

**Review by nikita-potashev — APPROVED** [review]

**Review by Priskorn — APPROVED** [review]

**Pull request merged**